### ISSUE TITLE
How to add message after redirect

### ISSUE THREAD

**Issue by shinokada** [Laravel-Backpack/PermissionManager#77]
![screen shot 2017-04-06 at 9 11 08 pm](https://cloud.githubusercontent.com/assets/147320/24753585/abe3ee16-1b0d-11e7-989f-16fbed44b99b.png)

I am wondering how to add a message like image when a user redirected after checked by PermissionManager.

**Comment by tabacitu** [comment]
Yup, you can use success, warning, danger and info. [Take a look here for details](https://github.com/prologuephp/alerts#adding-alerts-through-alert-levels). We use prologue/alerts for this.

Cheers!

**Comment by tabacitu** [comment]
@shinokada , thanks for bringing this up. I realised this Backpack/Base feature wasn't documented anywhere, so I wrote a basic usage now. [Check it out here to find out how to trigger alerts from javascript too](https://laravel-backpack.readme.io/docs/base).

Cheers!

**Comment by shinokada** [comment]
It seems you need `\Alert` rather than `Alert`

This one

    Alert::error(trans('cms.nopermission'))->flash();

gives an error.

    Class 'App\Http\Controllers\Admin\Alert' not found

But this one works.

    \Alert::error(trans('cms.nopermission'))->flash();

**Comment by tabacitu** [comment]
Hi @shinokada ,

Sure. Unless you ```use Alert``` at the beginning of your class, you'll need to use ```\Alert```.

Cheers!

**Comment by ccyeh0901** [comment]
Hi ~ my problem is when return view,  Alert works well, but if redirect to other page, 
the alert will not show, no matter flash() is added or not. 

Could you please inform any clue with this problem, thanks a lot~

**Comment by core45** [comment]
How can I do something like this?
return redirect('/admin/some_path/')->withErrors('Error message');
